src/runtime_src/core/tools/xbutil2/SubCmdValidate.cpp

```diff
@@ -857,6 +857,12 @@ dmaTest(const std::shared_ptr<xrt_core::device>& _dev, boost::property_tree::ptr
 
     std::stringstream run_details;
     size_t block_size = 16 * 1024 * 1024; // Default block size 16MB
+
+   // check if the bank has enough memory to allocate
+   // m_size is in KB so convert block_ize (bytes) to KB for comparision
```

Comment:
I suppose the xbutil1 checks this condition but xbutil2 doesn't, therefore there is an regression? Could you please explain the root cause of the fix and add into the comments of the CR?

Reply:
The memory bank got initially didn't have enough memory  for  the DMA test . So continue the loop till you get enough memory.